client/web/app/elements/lecture/editor.html

```diff
@@ -142,13 +140,8 @@
                 }
                 this.set('backendDocument', Config.urls.backend + '/' + this.lectureId);
             },
-            _lectureChanged: function(lecture) {
-                if (this.lecture.categories !== undefined && this.lecture.categories.length !== 0) {
-                    this.set('selectedCategory', this.lecture.categories[0]);
-                }
-            },
             _categorySelected: function(e, data) {
-                this.set('lecture.categories', [this.selectedCategory]);
+                this.set('lecture.category', data.item.dataValue);
```

Comment:
podwojnie ustawiana jest ta wartosc
1. selected="{{lecture.category}}"
2. this.set('lecture.category', data.item.dataValue);

jeden jest zbedny